src/components/home/elements/WithdrawOrDepositToken.vue

```diff
@@ -33,6 +33,41 @@
               {{ this.validContractType ? 'Invalid Contract ID' : 'Not PaymentChannel Contract or Lock Contract' }}
```

Comment:
Update word "Not PaymentChannel Contract or Lock Contract" to "This is not PaymentChannel Contract or Lock Contract"